act/io/__init__.py

```diff
@@ -34,5 +35,12 @@
             'read_psl_radar_fmcw_moment',
         ],
         'pysp2': ['read_hk_file', 'read_sp2', 'read_sp2_dat'],
+        'io_utils': ['pack_tar',
```

Comment:
@kenkehoe I think we need to move this over to the __init__ in the utils area as well.